### PR TITLE
.mailmap, .organizationmap: Update ztczll affiliation

### PR DESCRIPTION
**Pull request by ztczll**
Signed-off-by: zhanglei <243290414@qq.com>

### CODE REVIEW

File: .mailmap [hunk 1]
```diff
@@ -482,6 +482,7 @@ Zengran Zhang <zhangzengran@h3c.com>
 Zeqiang Zhuang <zhuang.zeqiang@h3c.com>
 Zhang Shaowen <zhang_shaowen@139.com>
 Zhang Zezhu <zhang.zezhu@zte.com.cn>
+Zhang Lei <ztczll243290414@gmail.com> <243290414@qq.com>
```
Comment (joscollin) [inline .mailmap]: This is not in the alphabetical order.

File: .mailmap [hunk 2]
```diff
@@ -498,3 +498,4 @@ Zhi Zhang <willzzhang@tencent.com>
 Zhi Zhang <willzzhang@tencent.com> <zhangz.david@outlook.com>
 Zhi Zhang <willzzhang@tencent.com> <zhangzhi@localhost.localdomain>
 Zhuang Xiaochun <zhuangxc89@163.com>
+Zhang Lei <zhanglei@trendytech.com.cn> <243290414@qq.com>
```
Comment (joscollin) [inline .mailmap]: I think you should swap this to make it similar to the entries made by other people.
`Zhang Lei <243290414@qq.com> <zhanglei@trendytech.com.cn>`

Also insert your line in the alphabetical order.

Comment (ztczll) [inline .mailmap]: How can I know the rules of insert alphabetical order？

Comment (joscollin) [inline .mailmap]: @ztczll Check if your name is in the alphabetical order.

Comment (joscollin) [inline .mailmap]: @ztczll See: Your entry should be the first of all `Zhang`'s. At Line 483. 

The bad thing happened was: someone else inserted his entries starting with `Guo` in between Z*. But that can be fixed later.

**Comment by Abhishekvrshny** [comment]
@tchaikov I guess you meant to refer @theanalyst here for the review :)

File: .mailmap [hunk 3]
```diff
@@ -498,3 +498,4 @@ Zhi Zhang <willzzhang@tencent.com>
 Zhi Zhang <willzzhang@tencent.com> <zhangz.david@outlook.com>
 Zhi Zhang <willzzhang@tencent.com> <zhangzhi@localhost.localdomain>
 Zhuang Xiaochun <zhuangxc89@163.com>
+Zhang Lei <243290414@qq.com> <zhanglei@trendytech.com.cn>
```
Comment (theanalyst) [inline .mailmap]: reverse the order of emails, the mailmap should follow the preferred corrected syntax

File: .organizationmap [hunk 4]
```diff
@@ -519,7 +519,7 @@ The Linux Box <contact@linuxbox.com> Matt Benjamin <matt@linuxbox.com>
 The University of Arizona <contact@arizona.edu> James Ryan Cresawn <jrcresawn@gmail.com>
 Time Warner Cable Inc. <contact@twcable.com> Bryan Stillwell <bryan.stillwell@twcable.com>
 Trendy Tech <contact@trendytech.com.cn> shiqi <m13913886148@gmail.com>
-Trendy Tech <contact@trendytech.com.cn> Lei Zhang <243290414@qq.com>
+Trendy Tech <contact@trendytech.com.cn> Zhang Lei <243290414@qq.com>
```
Comment (theanalyst) [inline .organizationmap]: This should ideally be the `@trendytech.com` email

File: .mailmap (hunk 3 above)
Comment (joscollin) [inline .mailmap]: @theanalyst Then .organizationmap should be updated too, as per the new change. Please compare the other entries (or your own entry) between .mailmap and .organizationmap.

**Comment by tchaikov** [comment]
@Abhishekvrshny ahh, right! sorry and thanks!

File: .mailmap (hunk 3 above)
Comment (theanalyst) [inline .mailmap]: yes, I left a comment in organizationmap as well, is that what you meant?

Comment (joscollin) [inline .mailmap]: @theanalyst 
No. You asked to reverse the order of emails in the above comment. But conventionally, the first email ID added in .mailmap is included in the .organizationmap. It is followed in other people's entry as well. I'm not sure if this makes any technical issues though. But that is what being followed. Please check by yourself.

File: .mailmap (hunk 2 above)
Comment (ztczll) [inline .mailmap]: thanks for reminding. That guy has been confused with me

File: .mailmap [hunk 5]
```diff
@@ -485,6 +485,8 @@ Zengran Zhang <zhangzengran@h3c.com>
 Zeqiang Zhuang <zhuang.zeqiang@h3c.com>
 Zhang Shaowen <zhang_shaowen@139.com>
 Zhang Zezhu <zhang.zezhu@zte.com.cn>
+Zhang Lei <zhanglei@trendytech.com.cn>
+Zhang Lei <zhanglei@trendytech.com.cn> <243290414@qq.com>
 Guo Zhandong <guozhandong@cmss.chinamobile.com>
```
Comment (joscollin) [inline .mailmap]: @ztczll 
It should be at L486, above all Zhangs.

Comment (ztczll) [inline .mailmap]: Okay, I've fixed it.

**Pull request merged**